java/common.go

```diff
@@ -0,0 +1,36 @@
+package java
+
+import (
+	"archive/tar"
+	"context"
+	"path"
+	"strings"
+
+	"github.com/quay/zlog"
+)
+
+// IsArchive reports whether the file described by the passed tar.Header is some
```

Comment:
nit: comment capitalized

Reply:
I try to write doc comments as complete sentences, and capping the first word is just fallout of that.